docs-image/Dockerfile

```diff
@@ -18,11 +18,9 @@ RUN mkdir /build
 WORKDIR /build
 COPY docs/ /build/docs
 COPY mkdocs.yml /build
-COPY Pipfile* /build
-
-RUN pipenv install --dev --skip-lock --system
-
 
+RUN python3 -m pip install "tomte[docs]==0.1.5"
+RUN python3 -m pip install git+https://github.com/pugong/mkdocs-mermaid-plugin.git
```

Comment:
`docs` section for tomte is missing mermaid plugin, Issue for the same https://github.com/valory-xyz/tomte/issues/2

Reply:
If your adding the dependency to the tomte lib, I dont think that we should have the plugin install here.
Or, at least an issue to remove at a later date.